includes/views/graceful-degradation.php

```diff
@@ -1,3 +1,3 @@
 <div class="notice notice-error is-dismissible">
-    <p><?php _e( 'Repeatable content mapping requires the installation of Advanced Custom Fields Pro WordPress Plugin. To enable this functionality, please install and refresh the page.', 'your-text-domain' ); ?></p>
+    <p><?php _e( 'Repeatable content mapping requires the installation of Advanced Custom Fields Pro WordPress Plugin. To enable this functionality, please install and refresh the page.', 'gathercontent-import' ); ?></p>
```

Comment:
```suggestion
    <p><?php _e( 'Repeatable content mapping requires the installation of Advanced Custom Fields Pro WordPress Plugin. To enable this functionality, please install and refresh the page.', 'content-workflow' ); ?></p>
```

Referencing the new name in the other PR: https://github.com/Bynder/cw-wordpress-plugin/pull/3/files#diff-42547caa37b3794ef2c07a534035a59df4409cf1b53283d2191142729385176f